### DISCUSSION TITLE
Pulse for Mac

### DISCUSSION THREAD

**Discussion by kean** [kean/Pulse#187] [Announcements]
Pulse for Mac app is currently available in TestFlight for testing: https://testflight.apple.com/join/1jcanE3q. This new app replaced Pulse Pro. It was completely redesigned and has a ton of new features, including response mocking and many more.

> Pulse for Mac supports earlier Pulse versions, but version 4.x is required to use the new features

**Comment by qyzdo** [comment]
Hi! Is the new version of the Pulse is gonna stay free? Or it will be paid app in the AppStore?

**Reply by kean** [reply]
I haven't decided on a pricing yet, but, yes, I'm considering making it a paid app.

**Reply by ahmadmssm** [reply]
If so, a one-time purchase would be very nice.

**Comment by JaviSoto** [comment]
It looks like the TestFlight link no longer works. Is there any other way to test it? 🙏

**Reply by kean** [reply]
Hey, @JaviSoto. Thanks for letting me know. I uploaded a new build to refresh the expiration time. I can't believe it's been 90 days. I've been too busy with the move to a new state and a new job, so I had zero time to work on this, unfortunately. 

_Update: it's in review_

**Reply by JaviSoto** [reply]
Thank you!

**Comment by alinless** [comment]
The Pulse client is no longer available on TestFlight. I'd really appreciate it if you could restore it

**Comment by RebelPoetry** [comment]
Still not available on TestFlight. I'd really appreciate it will appear. Thank you for your work!

**Reply by kean** [reply]
Try getting the latest version. I just tested 4.0.0 (13), and it seems to work (at least for me).

<img width="804" alt="Screenshot 2023-12-20 at 9 35 42 AM" src="https://github.com/kean/Pulse/assets/1567433/4ee8b02a-a9b8-4c78-9704-2f963e7be6dc">

**Reply by alinless** [reply]
@RebelPoetry, please try to reinstall the app. It helped me

**Reply by RebelPoetry** [reply]
Reinstall helped. Thank you!)

**Comment by Rashidium** [comment]
hey @kean great job. any chance we can try the app. test flight beta is full right now.

**Reply by kean** [reply]
Thanks for letting me know! I didn't expect it would hit the limit. I increased the limit a bit, so, please, try again.

**Comment by lukeredpath** [comment]
@kean the latest version on TestFlight seems to be crashing frequently to the point I can't really use it anymore - are you receiving crash reports?

**Reply by kean** [reply]
Hey, thanks for reporting it. I think you are right, there is some new crash related to the environment objects. I need a plan to migrate off of them.

<img width="769" alt="Screenshot 2024-07-11 at 8 26 33 AM" src="https://github.com/kean/Pulse/assets/1567433/7676ad3d-7ec2-4ad0-bcf3-6e73fd7b5231">

**Reply by kean** [reply]
It should be fixed in the latest version. Again, thanks for reporting it!
